src/main/modules/ardoq/IParser.ts

```diff
@@ -1,5 +1,5 @@
 import { Dependency } from './Dependency';
 
 export interface IParser {
-  extractTopTierDeps: (depString: string) => Dependency[];
+  extractTopTierDeps: (depString: string, depStringOther?: string | undefined) => Promise<Dependency[]>;
```

Comment:
I'm not seeing `depStringOther` used within the implementations of `extractTopTierDeps` ?

Reply:
It's not required for all implementations - it's an optional parameter here and in the HTTP JSON payload - it's only used for Yarn and NPM parsers